released/packages/coq-sail-stdpp/coq-sail-stdpp.0.19/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+maintainer: "Sail Devs <cl-sail-dev@lists.cam.ac.uk>"
+authors: [
+  "Alasdair Armstrong"
+  "Thomas Bauereiss"
+  "Brian Campbell"
+  "Shaked Flur"
+  "Jonathan French"
+  "Kathy Gray"
+  "Robert Norton"
+  "Christopher Pulte"
+  "Peter Sewell"
+  "Mark Wassell"
+]
+homepage: "http://www.cl.cam.ac.uk/~pes20/sail/"
+bug-reports: "https://github.com/rems-project/coq-sail/issues"
+doc: "https://github.com/rems-project/sail/blob/sail2/README.md"
+license: "BSD-3-clause"
+dev-repo: "git+https://github.com/rems-project/coq-sail.git"
+build: [
+  ["sh" "-exc" "./prepare-stdpp-src.sh"]
+  [make "-C" "src-stdpp"]
+]
+install: [make "-C" "src-stdpp" "install"]
+depends: [
+  "coq" {>= "8.18" & < "9.1~"}
```

Comment:
@bacam is it really known that this will work on 9.0.0? Have you tested? We can't test here yet since the opam package is not out.

Reply:
I built it (and a model using it) on 9.0+rc1 from the core-dev opam repository.  That said, I used the upper bound from stdpp!